packages/use-toasts/src/components/ScalarToasts.vue

```diff
@@ -42,18 +42,32 @@ initializeToasts((message, level = 'info', options: ToastOptions = {}) => {
   background: var(--scalar-background-1);
   color: var(--scalar-color-1);
   padding: 18px;
-  border-color: var(--scalar-background-3);
+  border: none;
```

Comment:
should we favor `transparent` for high contrast?

Reply:
Oh yeah that's a good idea, sorry didn't get notified about this for some reason